src/content/en/2020/seo.md

```diff
@@ -83,493 +83,603 @@ Response Code | All Devices | Mobile | Desktop
+In a move from ["strings to things"](https://blog.google/products/search/introducing-knowledge-graph-things-not/), search engines have agreed upon a broad vocabulary of objects in support of marking up a variety of people, places, and things within web content. However, only a subset of that vocabulary triggers inclusion within search results components. Google specifies those that they support and how they're displayed in their [search gallery](https://developers.google.com/search/docs/guides/search-gallery), and provides [a tool](https://search.google.com/test/rich-results) to validate their support and implementation. 
 
-As search engines evolve to reflect more of these elements in search results, the incidence rates of the different  vocabulary changes across the web.
+As search engines evolve to reflect more of these elements in search results, the incidence rates of the different vocabularies change across the web.
 
-As part of our examination, we took a look at the incidence rates of different types of structured markup. The available formats include [RDFa](https://www.w3.org/TR/rdfa-primer/) and [Schema.org](https://schema.org/) which come in both the microformats and [JSON-LD](https://www.w3.org/TR/json-ld11/) flavors. Google has recently [dropped the support for data-vocabulary](https://developers.google.com/search/blog/2020/01/data-vocabulary), a vocabulary that was primarily used to implement breadcrumbs. 
+{# TODO(authors): Is schema.org itself a "format"? #}
+As part of our examination, we took a look at the incidence rates of different types of structured markup. The available formats include [RDFa](https://www.w3.org/TR/rdfa-primer/) and [Schema.org](https://schema.org/), which come in both the microformats and [JSON-LD](https://www.w3.org/TR/json-ld11/) flavors. Google has recently [dropped the support for data-vocabulary](https://developers.google.com/search/blog/2020/01/data-vocabulary), which was primarily used to implement breadcrumbs. 
 
-JSON-LD is generally considered the more portable and easy to manage implementation and has become the preferred format. As a result, we see that JSON-LD appears on 29.78% of mobile pages and 30.60% of desktop pages.
+{# TODO(analysts, authors): Consider adding a "big number" figure here for the mobile JSON-LD stat, or a bar chart containing each format. #}
+JSON-LD is generally considered to be the more portable and easier to manage implementation and so it has become the preferred format. As a result, we see that JSON-LD appears on 29.78% of mobile pages and 30.60% of desktop pages.
 
+{# TODO(authors): Is this disparity really noteworthy? The difference seems quite small. #}
```

Comment:
@aleyda

Reply:
@rviscomi Although the disparity between mobile vs. desktop is not "big" it's important to show how there are still slightly more desktop pages featuring one, because of mobile first index. Would you suggest to eliminate the mention of the desktop instead?

Reply:
SGTM, I'll drop this